core/templates/pages/exploration-player-page/layout-directives/progress-nav.component.html

```diff
@@ -17,7 +17,7 @@
             (click)="validateIndexAndChangeCard(displayedCardIndex+1)"
             aria-label="Next Card"
             class="oppia-next-button e2e-test-next-button oppia-nav-button">
-      <i class="material-icons oppia-vcenter" aria-hidden="true">&#xE5C8;</i>
+      <i class=" oppia-vcenter" aria-hidden="true">&#xE5C8;</i>
```

Comment:
Hey @prafulbbandre, there seem to be an extra space here after `oppia-vcenter`, would you please remove that space. Thanks!

Reply:
yes sure.